services/Cargo.toml

```diff
@@ -66,13 +66,13 @@ strum = { version = "0.24", features = ["derive"] }
 time = "0.3"
 tokio = { version = "1.15", features = ["macros", "fs", "signal", "sync", "rt-multi-thread"] }
 tokio-util = { version = "0.7", features = ["codec"] }
-tonic = { version = "0.6", features = ["tls", "tls-roots"] }
+tonic = { version = "0.6", features = ["tls", "tls-roots"] } # must be compatible with `scienceobjectsdb_rust_api`
```

Comment:
couldn't they just reexport the dependency and us using that?

Reply:
Maybe. But this is the reason why we can't update it right now.